pkg/operator/deploy/deploy.go

```diff
@@ -138,6 +138,15 @@ func (o *operator) resources() ([]kruntime.Object, error) {
 		domain += "." + o.env.Domain()
 	}
 
+	ingressIP := o.oc.Properties.IngressProfiles[0].IP
+	if o.oc.Properties.IngressProfiles != nil && len(o.oc.Properties.IngressProfiles) > 1 {
```

Comment:
At this point we would have already blown up if `o.oc.Properties.IngressProfiles` was nil, perhaps we should invert the condition and return an error if it is nil and then proceed with the logic of using the first one or the one named `router-default`

Reply:
You're right lol! thanks, fixed